lib/ecto/repo.ex

```diff
@@ -64,9 +64,18 @@ defmodule Ecto.Repo do
       use the `:repo` property in the event metadata for distinguishing
       between repos.
 
-    * `:stacktrace`- when true, publishes the stacktrace in telemetry events
+    * `:stacktrace`- when `true`, publishes the stacktrace in telemetry events
       and allows more advanced logging.
 
+    * `:log_stacktrace_mfa` - A `{module, function, arguments}` tuple that customizes
+      which part of the stacktrace is included in query logs. The specified function
+      must accept at least two arguments (a stacktrace and the repo module) and return
+      a filtered stacktrace. Additional arguments can be passed in the third element
+      of the tuple. For `Ecto.Adapters.SQL`, defaults to
```

Comment:
```suggestion
    * `:log_stacktrace_mfa` - A `{module, function, arguments}` tuple that customizes
      which part of the stacktrace is included in query logs. The specified function
      must accept at least two arguments (stacktrace and metadata) and return
      a filtered stacktrace. The metadata is a map with keys such as `:repo` and other
      adapter specific information. Additional arguments can be passed in the third
      element of the tuple. For `Ecto.Adapters.SQL`, defaults to
```

Reply:
I propose one small change in case we need to pass more information in the future and you can merge it!